src/datadog.ts

```diff
@@ -11,6 +11,7 @@ import * as cdk from "@aws-cdk/core";
 import log from "loglevel";
 import { applyLayers, redirectHandlers, addForwarder, applyEnvVariables, defaultEnvVar } from "./index";
 import { Transport } from "./transport";
+const version = require("../version.json");
```

Comment:
I kept getting a ts error which came from trying to import the package.json file when it wasn't in the tsconfig rootdir

Reply:
Yeah, it's probably better to rely on version.json, since the package.json is node specific. Does importing json work with jsii/python support?

Reply:
I also just realized we don't build python in ci/cd. That would probably have answered my question

Reply:
Actually I think in the `test` job on the `build` workflow we run `yarn build` which compiles and packages the jsii packages. Check it out [here](https://github.com/DataDog/datadog-cdk-constructs/pull/34/checks?check_run_id=2415362957#step:6:38), JSII did not throw any errors saying importing json would break the Python artifact so I assume it works.